source/MaterialXGenShader/HwShaderGenerator.h

```diff
@@ -196,6 +196,7 @@ namespace HW
     extern const string USER_DATA_LIGHT_SHADERS;
 }
 
+/// Shader stage identifiers.
```

Comment:
This was removed in a previous change, in order to keep Doxygen from incorrectly presenting the Stage namespace as a class.

Reply:
I'll clean up.